astilectron.go

```diff
@@ -269,7 +274,7 @@ func (a *Astilectron) execute() (err error) {
 	} else {
 		singleInstance = "false"
 	}
-	var cmd = exec.CommandContext(ctx, a.paths.AppExecutable(), append([]string{a.paths.AstilectronApplication(), a.listener.Addr().String(), singleInstance}, a.options.ElectronSwitches...)...)
+	var cmd = exec.CommandContext(ctx, a.paths.AppExecutable(), append([]string{a.paths.AstilectronApplication(), a.listener.Addr().String(), singleInstance, a.options.AppName}, a.options.ElectronSwitches...)...)
```

Comment:
I'd rather have the full path provided here through the [Paths](https://github.com/asticode/go-astilectron/blob/master/paths.go#L13) structure:

1) create a new attribute named `electronUserData` in the `Paths` structure
2) compute its value in `newPaths` using the same logic you used in the JS PR. You should use other `Paths` attributes such as `electronDirectory` that could be useful in your case to build the value.
3) provide it to Electron the same way you provided `a.options.AppName`